src/main/java/org/kuali/kfs/gl/batch/service/impl/CuScrubberProcessImpl.java

```diff
@@ -309,4 +313,26 @@ protected TransactionError setupEntryWithPotentialContinuation(OriginEntryFull c
         return null;
     }
 
+    /**
+     * Overridden to also restrict plant indebtedness based on document type,
+     * by means of a custom PLANT_INDEBTEDNESS_DOCUMENT_TYPES parameter.
+     * 
+     * @see org.kuali.kfs.gl.batch.service.impl.ScrubberProcessImpl#processPlantIndebtedness(
+     * org.kuali.kfs.gl.businessobject.OriginEntryInformation, org.kuali.kfs.gl.service.ScrubberReportData)
+     */
+    @Override
+    protected String processPlantIndebtedness(OriginEntryInformation scrubbedEntry, ScrubberReportData scrubberReport) {
+        // Make sure the entry was from a document that supports plant indebtedness, similar to the logic from the processCapitalization() method.
+        ParameterEvaluator documentTypeCodes = (parameterService.getParameterValueAsBoolean(ScrubberStep.class,

```

Comment:
Some of the variable naming was based on that from the similar capitalization entry processing. However, I will work on modifying this to make it more readable. Thanks for the suggestions.